packages/terra-clinical-item-view/examples/test-examples/CommentItemView.example.jsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import ItemView from '../../lib/ItemView';
 
-const comment = <ItemView.Comment text="comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1" id="ItemComment" />;
+const comment1 = <ItemView.Comment text="comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1" id="ItemComment1" />;
```

Comment:
Instead of duplication, lets do:
```diff
const comment = (id) => (<ItemView.Comment id= {id} text="comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1" id="ItemComment" />);
- const comment1 = <ItemView.Comment text="comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1" id="ItemComment1" />;
- const comment2 = <ItemView.Comment text="comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1comment1" id="ItemComment2" />;
 
 const ItemViews = () => (
   <div style={{ width: '250px' }}>
     <h2>Default</h2>
-    <ItemView comment={comment1} id="ItemView1" />
+    <ItemView comment={comment(1)} id="ItemView1" />
     <br />
     <h2>Truncated</h2>
-    <ItemView isTruncated comment={comment2} id="ItemView2" />
+    <ItemView isTruncated comment={comment(2)} id="ItemView2" />
   </div>
 );
 

```

Reply:
I tired that. It won't pass the axe tests. `ItemComment` will still be the common id to both comments.

Reply:
If we remove `id="ItemComment"` at the end of @emilyrohrbough's `const comment`, the id's will truly be unique based on the `id` param. Do the axe tests still fail with that?

Reply:
Oops. That should work.

https://github.com/cerner/terra-clinical/pull/304/commits/f5ca940df5963f603c8a0f7ba5216891e9453963